introduction-to-datasets.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
+
+The default storage _scheme_ (as opposed to a _[schema](#schemas)_, described below) is Hive. If you omit the scheme in your URI, your dataset is stored in Hive. For example, the URI `movies` is equivalent to `dataset:hive:/movies`. A dataset you create with this URI is stored as a table in Hive's `default` database.
+
+Dataset URIs cannot contain periods (.).
```

Comment:
Dataset names and namespaces can't contain periods for compatibility with Avro and Hive. Other parts of the URI can.

It would be better to state that we put a restriction on the name and namespace parts of the URI, which are used when the underlying system (like Hive) supports them. Names and namespaces must be made of alphanumeric characters or underscore and can't start with a number. I think record field names have the same requirement.